### ISSUE TITLE
2.0.1 with Ti SDK 5.2.0.GA gives a black screen

### ISSUE THREAD

**Issue by yozef** [hansemannn/Ti.GoogleMaps#8]
I was trying to use a version of Ti.GoogleMaps (2.1.0) where I have made some updates to the Module with Ti.SDK 5.2.0.GA and it gave a black screen, I thought it was because of my changes, so I've used the last stable version 2.0.1 with Ti SDK 5.2.0.GA (which came out today), and I still got the black screen. If I use Ti SDK 5.1.2.GA, it works, it's only with 5.2.0.GA where the black screen is happening.

Here is what the console gave:
~~~
[ERROR] :  An error occurred during build after 29s 730ms
[ERROR] :  Detected multiple crashes:
[ERROR] :    /Users/user/Library/Logs/DiagnosticReports/TiGoogleMapsAdditions_2016-02-23-114522_iMac5K.crash
[ERROR] :    /Users/user/Library/Logs/DiagnosticReports/TiGoogleMapsAdditions_2016-02-23-115116_iMac5K.crash
[ERROR] :    /Users/user/Library/Logs/DiagnosticReports/TiGoogleMapsAdditions_2016-02-23-115229_iMac5K.crash
[ERROR] :    /Users/user/Library/Logs/DiagnosticReports/TiGoogleMapsAdditions_2016-02-23-115604_iMac5K.crash
[ERROR] :  Note: these crashes may or may not be related to running your app.
~~~

<img width="243" alt="screen shot 2016-02-23 at 12 54 21 pm" src="https://cloud.githubusercontent.com/assets/287256/13261104/9f6f0102-da2c-11e5-9178-a67e77e9032a.png">

It's a shame, I just finished adding some features to Ti.GoogleMaps.

**Comment by hansemannn** [comment]
Hey Yosef, I had the same when testing the module in Xcode directly (its related to the bundle). Did everything work in 2.0.0?

**Comment by yozef** [comment]
Hey @hansemannn ti.googlemaps 2.0.1 with Ti SDK 5.2.0.GA gave the same issue. Not sure if the issue is from Ti 5.2.0 or the module.

**Comment by hansemannn** [comment]
Thanks man! It's related to the storyboard changes of 5.2.0.GA (https://jira.appcelerator.org/browse/TIMOB-19694). We need to investigate what's the best solution, I will keep this issue open and updated for updates on both sides.

**Comment by abada** [comment]
> EDIT: A workaround could be to copy the GoogleMaps.bundle inside the platform/ios directory again, until we find a better solution

Is this solution for `TISDK5.2.0.GA` ?

**Comment by hansemannn** [comment]
Just troubleshooting. Here is the follow-up ticket: https://jira.appcelerator.org/browse/TIMOB-20489

**Comment by hansemannn** [comment]
PR to be merged as soon as the above issue is resolved and 5.2.1.GA is released: https://github.com/hansemannn/Ti.GoogleMaps/pull/13

Before that, I will add a notice to the README.md on how to fix it manually.

**Comment by hansemannn** [comment]
Fixed in https://github.com/hansemannn/Ti.GoogleMaps/pull/13, including changes in the README.md. Thanks for reporting!
